dist/ContentPlaceholderPlugin.js

```diff
@@ -156,19 +156,16 @@ var ContentPlaceholderView = function () {
     key: '_show',
     value: function _show() {
       var el = this._el;
-      if (el && this._visible !== true) {
-        this._visible = true;
```

Comment:
Is there any good reason that you want to drop `this._visible` and use `el.style.display` instead?

Usually it's relatively inefficient and unsafe to use DOM element as source of truth for state management. 

Ideally DOM element should only be used as a pure view that reflects the managed state.

Thought this change does not change the behavior, it does change the idea behind the implementation. 

Unless you have strong reason that this should change, I'd recommend reverting this file.

Reply:
I was running into a race condition where the editor state would be from the previous comment, and `_visible` would be set to `false` when the placeholder was still shown.

Screen recording of the bug (placeholder still shown while editing second comment):
![comment-placeholder-bug](https://user-images.githubusercontent.com/1837091/53997100-c7811e00-40ef-11e9-8a2a-d0e08b3deb8c.gif)